packages/core-api/src/lib/AuthSessionManager/SamlAuthSessionManager.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { SessionManager, GetSessionOptions } from './types';
+import {
+  SamlAuthConnector,
+  SamlResponse,
+} from '../AuthConnector/SamlAuthConnector';
+import { SessionStateTracker } from './SessionStateTracker';
+
+type Options = {
+  connector: SamlAuthConnector<SamlResponse>;
+};
+
+export class SamlAuthSessionManager<T> implements SessionManager<T> {
```

Comment:
Is this the same as the original? These helper classes are intended to be generic and use-case oriented. Only copy if you need them to support something new, and in the case name it accordingly